### ISSUE TITLE
Deprecation warnings from Base.ASCIIString in Julia v0.5

### ISSUE THREAD

**Issue by elnjensen** [iancze/DiskJockey#20]
When running any DiskJockey command using Julia 0.5, many deprecation warnings about `String` vs. `Base.ASCIIString` are generated:

```
DJ_plot_chmaps.jl --linear
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:65
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:67
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:67
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:75
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:141
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:141
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:145
WARNING: Base.ASCIIString is deprecated, use String instead.
  likely near /Users/ejensen1/.julia/v0.5/DiskJockey/src/constants.jl:145
```

If this is as simple as replacing all instances of `ASCIIString` with `String` then feel free to assign it to me.

**Comment by elnjensen** [comment]
If starting Julia from the command line, one can use `julia --depwarn=no`, but if calling individual scripts, one would have to edit each script.  Plus, any code that uses the `NPZ` package generates the same warning from that package.

**Comment by elnjensen** [comment]
As a side note on this issue, the version of `NPZ` installed from the Julia package repository also gives similar deprecation warnings (when included from within DJ) but the latest version on Github works fine with Julia v0.5 with no warnings.  So doing:

```
julia> Pkg.rm("NPZ")
```
followed by 

```
julia> Pkg.clone("https://github.com/fhs/NPZ.jl")
```

will get the latest version and remove the warnings.

**Comment by iancze** [comment]
Ah, thanks for the fix. Hopefully the main branch of the NPZ package will be updated soon, so I can continue to use the more general package management syntax.

**Comment by iancze** [comment]
Hi Eric, 

The latest version of NPZ, as well as the new version of DiskJockey (available via the master branch, or the tag v0.1.4) should have fixed most of these warnings. I'm going to close this for now but please re-open this issue if they persist. 

Best,

Ian

**Comment by elnjensen** [comment]
Thanks, Ian, for this and other fixes!
